bodo/pandas/series.py

```diff
@@ -3072,12 +3061,20 @@ def sig_bind(name, accessor_type, *args, **kwargs):
         ("axis", inspect.Parameter.KEYWORD_ONLY, (None,)),
         ("inplace", inspect.Parameter.KEYWORD_ONLY, (False,)),
     ],
-    "str.replace": [
+    "replace": [
         ("to_replace", inspect.Parameter.POSITIONAL_OR_KEYWORD, (None,)),
         ("value", inspect.Parameter.POSITIONAL_OR_KEYWORD, (None,)),
         ("regex", inspect.Parameter.KEYWORD_ONLY, (False,)),
         ("inplace", inspect.Parameter.KEYWORD_ONLY, (False,)),
```

Comment:
What about the `limit` and `method` arguments for Series.replace? We don't support them yet?